system_test/common/aest_db_SUITE.erl

```diff
@@ -28,14 +31,29 @@
 -define(MINING_TIMEOUT,   3000).
 -define(GRACEFUL_STOP_TIMEOUT, 60000).
 
+%=== RECORDS ===================================================================
+
+-record(db_reuse_test_spec, {create,     % Node spec.
+                             populate =  % DB insertion.
+                                 fun populate_db/2,
+                             pre_reuse = % DB transformation.
+                                 fun(_,_,_) -> ok end,
+                             reuse,      % Node spec.
+                             assert =    % DB assertion.
+                                 fun assert_db_reused/3
+                            }).
+
 %=== COMMON TEST FUNCTIONS =====================================================
 
 all() -> [
           node_can_reuse_db_of_other_node,
           roma_node_can_reuse_db_of_other_roma_node,
           minerva_node_with_epoch_db_can_reuse_db_of_roma_node,
           node_can_reuse_db_of_roma_node,
-          node_can_reuse_db_of_minerva_node_with_epoch_db
+          node_can_reuse_db_of_minerva_node_with_epoch_db,
+          minerva_node_with_channels_update_as_tuple_can_reuse_db_of_analogous_node,
+          minerva_node_with_channels_update_as_tuple_can_reuse_db_of_analogous_node_with_force_progress_tx,
+          node_can_reuse_db_of_minerva_node_with_channels_update_as_tuple_with_force_progress_tx
```

Comment:
Fails with function clause (I did not manage to get a more details error message) on aeternity/aeternity Docker image d51d8bafef35 i.e. commit a2fdf3bfc i.e. tag v3.0.0

Passes on Docker image aeternity/aeternity:master 164be403961a i.e. commit d67843a9f

Reply:
A more verbose error message than function clause would be possible by amending test with mining of the tx on chain. Though that would require spending further time. Patches welcome :) I consider this level of error checking good enough.